src/forms/StudyFormSteps.js

```diff
@@ -259,56 +301,111 @@ export const GrantStep = ({
         handleChange={handleChange}
         handleBlur={handleBlur}
         handleFocus={id => setFocused(id)}
+        readOnly={!editing && !newStudy}
       />
-      <FormField
-        id="description"
-        name="Description"
-        description="Study description in markdown, commonly the X01 abstract
+      {!editing && !newStudy ? (
+        <>
+          <Header as="h5">Description:</Header>
+          <Segment
+            attached={values.description.length > 0}
+            className={foldDescription ? 'max-h-500' : null}
+          >
+            {values.description && values.description.length > 0 ? (
+              <Markdown
+                source={values.description}
+                renderers={{
+                  image: Image,
+                  table: props => <Table>{props.children}</Table>,
+                  list: List,
+                  listItem: List.Item,
+                }}
+              />
+            ) : (
+              <Header disabled className="mt-6" as="h5">
+                No study description available.
+              </Header>
+            )}
+          </Segment>
+          {values.description && values.description.length > 0 && (
```

Comment:
Looks like the `Read More` length shoudl be set to around **170-175** characters and the `Read Less` functionality doesn't expand the height of the text area or wrap the text 
<img width="1104" alt="Screen Shot 2019-09-24 at 4 24 12 PM" src="https://user-images.githubusercontent.com/1334131/65547679-f625e100-dee7-11e9-83c9-28a26be36f6a.png">
<img width="1179" alt="Screen Shot 2019-09-24 at 4 24 23 PM" src="https://user-images.githubusercontent.com/1334131/65547685-f7efa480-dee7-11e9-93eb-e5d01c0e0a9b.png">